bin/commands/cmd_gen.ml

```diff
@@ -30,7 +30,35 @@ let rec atdgen ?(rescript = false) ?output_prefix:original_prefix = function
     | 0, i | i, _ ->
       Error (`Atd_error (i, file)))
 
-let run ?output_prefix ?(rescript = false) files =
+let rec checkout ~git_file = function
+  | [] ->
+    Ok []
+  | file :: tail ->
```

Comment:
That's right, bad naming on my part. I'll rename it to `head`. (I think it will then be clear from the context and not confusing anymore, otherwise I'll go for something more verbose.)